README.md

```diff
@@ -117,6 +117,10 @@ Here are the instructions you should follow to replicate my AwesomeWM setup.
    - You will need to create the directory if it does not exist.
    - It does not matter that the actual font files (`.ttf`) are deep inside multiple directories. They will be detected as long as they can be accessed from `~/.fonts` or `~/.local/share/fonts`.
 
+   You can find the fonts required inside the fonts folder in misc
+   ```shell
+   cp -r ./misc/fonts/* ~/.fonts/  [ or to ~/.local/share/fonts/]
```

Comment:
```suggestion
   cp -r ./misc/fonts/* ~/.fonts/
   # Or to ~/.local/share/fonts
   cp -r ./misc/fonts/* ~/.local/share/fonts/
```
Small change here: I would prefer every command to be in its own line, so that it is easily copy-pastable.